src/main/java/domen/rideapp/infrastructure/AppConfig.java

```diff
@@ -38,4 +57,47 @@ RideService rideService(PricingService pricingService,
     DriverService driverService(DriverRepository repository) {
         return new DriverService(repository);
     }
+
+    @Bean
+    @ConfigurationProperties(prefix = "google.maps")
+    public GoogleMapsConfig googleMapsConfig() {
```

Comment:
Masz tutaj dużo konfiguracji z googlem zależne od profili, config properties. Zalecam stworzenie osobnej klasy konfiguracyjnej pod google. Dodatkowo zauważ że niektóre beany masz publiczne a niektóre default-package, ustandaryzuj to